RoboFile.php

```diff
@@ -70,15 +74,42 @@ public function testFull(string $site_name = NULL) {
     // Test all the Solr things.
     $this->testSolrEnabled($site_name);
 
+    // Test creating Solr index.
+    $this->testSolrIndexCreate($site_name, 'dev');
+
+    // Test select query.
+    $this->testSolrSelect($site_name, 'dev');
+
+    // Finally, run Solr diagnose.
+    $this->testSolrDiagnose($site_name, 'dev');
+
     $this->output()->write( 'All done! 🎉' );
     return ResultData::EXITCODE_OK;
   }
 
   /**
    *
    */
-  protected function getCurrentBranch(): ?string {
-    return trim(shell_exec('git rev-parse --abbrev-ref HEAD'));
+  protected function getCurrentConstraint(): string {
```

Comment:
Nit: It would be good if we added some function docs rather than empty comment blocks.

Reply:
I'm going ahead and merging this but definitely will add more docs in my next PR. Thanks!